### ISSUE TITLE
scripting with codespell: how to ascertain by exit code if codespell successfully or unsuccessfully found entry

### ISSUE THREAD

**Issue by luzpaz** [codespell-project/codespell#2440]
We can test if words exist in codespell using this one liner:  
![Screenshot_20220806_162815](https://user-images.githubusercontent.com/4140247/183265129-c22af606-23ee-4315-9bdd-1884474bff53.png)

I'd like to know if codespell can be queried in a script and tested by exit code if a query succeeds or not.

**Comment by peternewman** [comment]
Yes, the programme exit code will tell you whether typos were present or not.
